src/main/java/io/kestra/plugin/aws/glue/StopJobRun.java

```diff
@@ -0,0 +1,181 @@
+        )
+    }
+)
+public class StopJobRun extends AbstractGlueTask implements RunnableTask<Output> {
+
+    @Schema(title = "The name of the Glue job to stop")
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> jobName;
+
+    @Schema(title = "The ID of the job run to stop")
+    @PluginProperty(dynamic = true)
+    @NotNull
+    private Property<String> jobRunId;
+
+    @Schema(
+        title = "Wait for the job to be fully stopped before ending the task.",
+        description = "If true, the task will periodically check the job status until it reaches a stopped state."
+    )
+    @Builder.Default
+    private Property<Boolean> wait = Property.ofValue(true);
+
+    @Schema(
+        title = "Interval between status checks"
+    )
+    @Builder.Default
+    private Property<Duration> interval = Property.ofValue(Duration.ofSeconds(1));
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        // Render properties once
+        String jobNameValue = runContext.render(this.jobName).as(String.class).orElseThrow();
+        String jobRunIdValue = runContext.render(this.jobRunId).as(String.class).orElseThrow();
+        boolean waitValue = runContext.render(this.wait).as(Boolean.class).orElse(true);
+        Duration intervalValue = runContext.render(this.interval).as(Duration.class).orElse(Duration.ofSeconds(1));
+
+        try (GlueClient glueClient = this.client(runContext)) {
+            // Stop the job
+            BatchStopJobRunResponse stopResponse = glueClient.batchStopJobRun(
+                BatchStopJobRunRequest.builder()
+                    .jobName(jobNameValue)
+                    .jobRunIds(jobRunIdValue)
+                    .build()
+            );
+
+            runContext.logger().info("Stopped Glue job '{}' with run ID: {}", jobNameValue, jobRunIdValue);
+
+            GetJobRunRequest getJobRunRequest = createGetJobRunRequest(jobNameValue, jobRunIdValue);
+            AtomicReference<software.amazon.awssdk.services.glue.model.JobRun> currentJobRun = new AtomicReference<>();
+
+            GetJobRunResponse initialResponse = glueClient.getJobRun(getJobRunRequest);
+            software.amazon.awssdk.services.glue.model.JobRun initialJobRun = initialResponse.jobRun();
+            currentJobRun.set(initialJobRun);
+
+            runContext.logger().info("Initial job state after stop request: {}", initialJobRun.jobRunStateAsString());
+
+            // Wait for job to reach terminal state
+            if (waitValue) {
+                waitForJobStopped(runContext, glueClient, getJobRunRequest, currentJobRun, intervalValue);
+            }
+
+            // Check state
+            var acceptableStopStates = Set.of(
+                software.amazon.awssdk.services.glue.model.JobRunState.STOPPED,
+                software.amazon.awssdk.services.glue.model.JobRunState.SUCCEEDED,
+                software.amazon.awssdk.services.glue.model.JobRunState.FAILED
```

Comment:
I see there's other potential stop states [in the doc](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/glue/model/JobRunState.html):

ERROR 
EXPIRED 
TIMEOUT 

What do you think?